class.json-api-endpoints.php

```diff
@@ -1084,14 +1101,16 @@ function get_media_item_v1_1( $media_id ) {
 
 		if ( in_array( $ext, array( 'mp3', 'm4a', 'wav', 'ogg' ) ) ) {
 			$metadata = wp_get_attachment_metadata( $media_item->ID );
+			$response['length'] = $metadata['length'];
 			$response['exif']   = $metadata;
 		}
 
 		if ( in_array( $ext, array( 'ogv', 'mp4', 'mov', 'wmv', 'avi', 'mpg', '3gp', '3g2', 'm4v' ) ) ) {
 			$metadata = wp_get_attachment_metadata( $media_item->ID );
-			if ( isset( $metadata['height'], $metadata['width'] ) ) {
+			if ( isset( $metadata['height'], $metadata['width'], $metadata['length'] ) ) {
```

Comment:
Should this `isset` check be back-ported to WPCOM? Also, I'd expect to see it guarding each property separately, or at least separating dimensions (height/width) from length.